src/molecules/LoginForm/reducer.ts

```diff
@@ -0,0 +1,43 @@
+import { InputTypes } from 'context';
+
+export enum LoginFormActions {
+  checkbox = 'TOGGLE CHECKBOX',
+  input = 'SET INPUT',
+}
```

Comment:
```
export enum LoginFormActions {
  TOGGLE_CHECKBOX = 'TOGGLE_CHECKBOX',
  SET_INPUT_VALUE = 'SET_INPUT_VALUE',
}
```
I think it will be better. Actions should have clear names (that tells you what the action do).